integration-tests/cli/test/metadata.test.ts

```diff
@@ -126,3 +126,18 @@ test.serial('does not log credentials', async (t) => {
   t.regex(logString, /(pin_number)/i);
   t.notRegex(logString, new RegExp(sensitiveValue), 'i');
 });
+
+// Test for auto-installing adaptor
```

Comment:
Can we add a line of comment along the lines of 

> This test should auto-install the common adaptor but then happily fail to generate metadata

It's a slightly strange test, conceptually, and I just want to clarify that we don't expect any metadata output